packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -211,445 +192,285 @@ void createSettingsAarGradle(Directory androidDirectory) {
   // Copy the new file.
```

Comment:
Please make the *** message above prettier (using Unicode box drawing characters or something).

Reply:
Done :)